README.md

```diff
@@ -1,199 +1,16 @@
-* VCX requires some payment plugin.
-[Here](https://github.com/hyperledger/indy-sdk/tree/master/libnullpay/README.md) is the simple plugin that can be used.
-
-### Ubuntu based distributions (Ubuntu 16.04 and 18.04)
-It is recommended to install the VCX packages with APT:
-
-    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
-    sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb (xenial|bionic) {release channel}"
-    sudo apt-get update
-    sudo apt-get install -y libvcx
-
-* (xenial|bionic) xenial for 16.04 Ubuntu and bionic for 18.04 Ubuntu.
-* {release channel} must be replaced with master, rc or stable to define corresponded release channel.
-Please See the section [Release channels](../README.md/#release-channels) for more details.
-
-### Windows
-
-1. Go to https://repo.sovrin.org/windows/libvcx/{release-channel}.
-2. Download last version of libvcx.
-3. Unzip archives to the directory where you want to save working library.
-4. After unzip you will get next structure of files:
-
-* `Your working directory`
-    * `include`
-        * `vcx.h`
-    * `lib`
-        * `vcx.dll`
-        * `nullpay.dll`
-        * `indy.dll`
-        * `libeay32md.dll`
-        * `libsodium.dll`
-        * `libzmq.dll`
-        * `ssleay32md.dll`
-
-`include` contains c-header file which contains all necessary declarations
-that may be need for your applications.
-
-`lib` contains all necessary binaries which contains libvcx and all it's dependencies.
- `You must add to PATH environment variable path to lib`. It's necessary for dynamic linkage
- your application with libvcx.
-
-{release channel} must be replaced with master, rc or stable to define corresponded release channel.
-
-#### Centos
-1. Go to https://repo.sovrin.org/rpm/libvcx/{release-channel}.
-2. Download and unzip the last version of library.
-3. Install with `rpm -i libvcx-version.rpm`.
-
-### MacOS
-
-1. Go to `https://repo.sovrin.org/macos/libvcx/{release-channel}`.
-2. Download the latest version of libvcx.
-3. Unzip archives to the directory where you want to save working library.
-4. After unzip you will get next structure of files:
-
-* `Your working directory`
-    * `include` - contains c-header files which contains all necessary declarations that may be need for your applications.
-        * `...`
-    * `lib` - contains library binaries (static and dynamic).
-        * `libvcx.a`
-        * `libvcx.dylib`
-    
-You need add the path to lib folder to LIBRARY_PATH environment variable. 
-    
-{release channel} must be replaced with master, rc or stable to define corresponded release channel.
+* VCX requires access to some [Mediator Agency](https://github.com/AbsaOSS/vcxagencynode) 
+* VCX requires some [payment plugin](https://github.com/hyperledger/indy-sdk/tree/master/libnullpay/README.md) is the simple plugin that can be used.
```

Comment:
I don't think this is really useful without more information about what the plugin is for, or at least a link to [this](https://hyperledger-indy.readthedocs.io/projects/sdk/en/latest/docs/design/004-payment-interface/README.html)